netilion/client.py

```diff
@@ -418,3 +419,12 @@ def patch_json_attachment(self, attachment: dict, attachment_id: int, attachment
             raise MalformedNetilionApiRequest(response)
         else:
             self.logger.info(f"PATCH confirmed: {response.status_code}")
+
+    def get_node_assets(self, node_id: int) -> list[Asset]:
+        url = self.construct_url(self.ENDPOINT.NODE_ASSETS, {"node_id": node_id})
+        response = self.get(url)
+        if response.status_code >= 300:
```

Comment:
Maybe use exact status code for success

Reply:
I updated the code to check for success first and handle errors in the else block